lib/main.dart

```diff
@@ -202,10 +202,10 @@ class _AppState extends State<App> with TickerProviderStateMixin {
 
 class _ScrollBehavior extends ScrollBehavior {
   @override
-  Widget buildViewportChrome(BuildContext context, Widget child, AxisDirection axisDirection) {
+  Widget buildOverscrollIndicator(BuildContext context, Widget child, ScrollableDetails details) {
     final theme = Theme.of(context);
```

Comment:
In a separate pull request, we could probably put the strantching overscoll here from the API 31. Let me know what you think, I'm not sure about it https://github.com/nt4f04uNd/sweyer/issues/96